var/spack/repos/builtin/packages/py-typing-extensions/package.py

```diff
@@ -28,7 +29,7 @@ class PyTypingExtensions(PythonPackage):
     version("3.6.6", sha256="51e7b7f3dcabf9ad22eed61490f3b8d23d9922af400fe6656cb08e66656b701f")
 
     # typing-extensions 4+ uses flit
-    depends_on("python@3.7:", when="@4.2:", type=("build", "run"))
```

Comment:
This is one of those deps of a build dep that we're supposed to keep supporting on 3.6. Can you re-add this line and maybe add a comment explaining why we need to keep it for now?

Reply:
Done.